exonum-java-binding-core/rust/src/storage/core_schema.rs

```diff
@@ -94,3 +94,25 @@ pub extern "system" fn Java_com_exonum_binding_blockchain_CoreSchemaProxy_native
     });
     utils::unwrap_exc_or(&env, res, ptr::null_mut())
 }
+
+/// Returns the configuration for the latest height of the blockchain. Throws
+/// `java.lang.RuntimeException` if the "genesis block" has not been created yet.
+#[no_mangle]
+pub extern "system" fn Java_com_exonum_binding_blockchain_CoreSchemaProxy_nativeGetActualConfiguration(
+    env: JNIEnv,
+    _: JClass,
+    schema_handle: Handle,
+) -> jstring {
+    let res = panic::catch_unwind(|| {
+        let val = match utils::cast_handle::<SchemaType>(schema_handle) {
+            SchemaType::SnapshotSchema(schema) => schema.actual_configuration(),
+            SchemaType::ForkSchema(schema) => schema.actual_configuration(),
+        };
+        serde_json::to_value(val)
```

Comment:
This is redundant, you should be able to use `serde_json::to_string` directly, without conversion to `serde_json::Value`

Reply:
Thanks, corrected